fastestimator/network.py

```diff
@@ -27,9 +27,11 @@
 class BaseNetwork:
     def __init__(self, ops: Iterable[Union[TensorOp, Scheduler[TensorOp]]]):
         self.ops = to_list(ops)
+        self.models = to_list(_collect_models(ops))
```

Comment:
`
@property
def models(self):
  return to_list(_collect_models(ops))
`
that way we don't store redundant information, and if op list is updated somehow models will still be correct

Reply:
op list won't be updated so I think we are fine there.